Remotion/Core/Core/Logging/Log4NetTraceListener.cs

```diff
@@ -206,22 +206,22 @@ public override void TraceData (TraceEventCache eventCache, string source, Trace
     ///     path='Log4NetTraceListener/Trace/param[@name="eventCache" or @name="source" or @name="id" or @name="message"]' />
     /// <include file='..\doc\include\Logging\Log4NetTraceListener.xml' path='Log4NetTraceListener/TraceTransfer/param[@name="relatedActivityId"]' />
     /// <include file='..\doc\include\Logging\Log4NetTraceListener.xml' path='Log4NetTraceListener/Trace/remarks' />
-    public override void TraceTransfer (TraceEventCache eventCache, string source, int id, string message, Guid relatedActivityId) 
+    public override void TraceTransfer (TraceEventCache? eventCache, string source, int id, string? message, Guid relatedActivityId)
     {
       TraceEvent (eventCache, source, TraceEventType.Information, id,
           message + ", relatedActivityId=" + relatedActivityId, new object[0]); 
```

Comment:
Should we do another TODO here to concat/log a sensible default value instead of empty string?